### PR TITLE
SBS-3 a + b

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: Count.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?php
+
```
Comment (mpaz-redstage) [inline Count.php:2]: remove extra space

File: Count.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+$a = 1;
+$b = 2;
+
+echo $a + $b;
```
Comment (mpaz-redstage) [inline Count.php:6]: Insert a new line at the end of file

File: Count.php (hunk 1 above)
Comment (mauriciopazpp) [reply Count.php:2]: :+1:

**Review by mauriciopazpp — APPROVED** [review]

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
!

**Pull request closed without merging**